wire-runtime/src/jsMain/kotlin/com/squareup/wire/internal/-Platform.kt

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2019 Square Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.squareup.wire.internal
+
+import okio.IOException
+import kotlin.reflect.KClass
+
+actual interface Serializable
+
+actual annotation class Throws(actual vararg val exceptionClasses: KClass<out Throwable>)
```

Comment:
Use https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-optional-expectation/index.html to avoid actuals in other platforms